File: Project_Files/dog/PyTest/dog.py
class Dog:
    def __init__(self, the_owner, new_name, new_favorite_food, new_breed, new_gender):
        self.my_owner = the_owner
        self.name = new_name
        self.favorite_food = new_favorite_food
        self.breed = new_breed
        self.gender = new_gender

    def __str__(self):
        return f'{self.name} ({self.gender}) the {self.breed} who likes to eat {self.favorite_food}'

File: Project_Files/Tests/JavaScript_Dog_Python_Bloodbank/model_answer/MochaChai/src/Dog.js
class Dog {
	constructor (theDogOwner, newName, newBreed, newGender, newFavoriteFood) {
    this.myDogOwner = theDogOwner
    this.name = newName
    this.breed = newBreed
    this.gender = newGender
    this.favoriteFood = newFavoriteFood
  }
  toString () {
  	return this.name + ' (' + this.gender + ') the ' + this.breed + ' who likes to eat ' + this.favoriteFood
  }
}